gocd/templates/pipelines/taskbroker.libsonnet

```diff
@@ -0,0 +1,78 @@
+local gocdtasks = import 'github.com/getsentry/gocd-jsonnet/libs/gocd-tasks.libsonnet';
+
+local checks_stage = {
+  checks: {
+    fetch_materials: true,
+    jobs: {
+      checks: {
+        timeout: 1200,
```

Comment:
Just a heads up, this is in minutes not seconds. We recently fixed this for getsentry.